lib/ansible/modules/cloud/docker/docker_swarm.py

```diff
@@ -136,6 +136,13 @@
     - docker
 requirements:
     - python >= 2.7
+    - docker-py >= 1.7.0
```

Comment:
I think `docker_swarm` is for docker's built-in swarm mode, `docker-py` should have a higher version number?

Reply:
Per http://docker-py.readthedocs.io/en/stable/change-log.html, it should be >= `1.10.0`

Reply:
Hmm, good question; the code also uses the `log_driver` parameter of `create_swarm_spec`, which has (according to the change log) only introduced in 2.6.0; I'll bump it up to that for now. @tbouvet: do you know more details?